convert_kinbody_to_urdf.py

```diff
@@ -0,0 +1,194 @@
+# convert_kinbody_to_urdf.py
+
+import os
+import sys
+import xml.etree.ElementTree as ET
+
+
+class Kinbody_Converter:
+    def __init__(self):
+        self.kinbody = None
+        self.urdf = None
+        self.link = None
+
+    def parse_from_file(self, filename):
+        if not filename.endswith('.kinbody.xml'):
+            print('Invalid input file: %s' % filename)
+            return False
+
+        self.kinbody = ET.parse(filename)
+        if self.kinbody is None:
+            print('Could not open file: %s' % filename)
+            return False
+
+        print('read kinbody from  %s' % filename)
+        return True
+
+    # TODO:
+    # handle Quat, RotationAxis
+    # handle sphere with Render and 0 radius
```

Comment:
I added the converting script for sphere with Render and 0 radius in [commit `bb4b8cd`](https://github.com/personalrobotics/pr-ordata/pull/22/commits/bb4b8cdc5232c5187b7e16fdc9892f60ff90f7ef)